server/database/users.go

```diff
@@ -3,21 +3,23 @@ package database
 import (
 	"database/sql"
 	"errors"
+	"github.com/gdg-garage/space-tycoon/server/stycoon"
 )
 
 func InsertUser(db *sql.DB, username string, password string) error {
 	_, err := db.Exec(`insert into d_user (name, password) values (?, ?)`, username, password)
 	return err
 }
 
-func GetUserPassword(db *sql.DB, username string) (string, error) {
+func GetUserPassword(db *sql.DB, username string) (stycoon.PlayerId, string, error) {
 	var password sql.NullString
-	err := db.QueryRow("select password from d_user where name = ? limit 1", username).Scan(&password)
+	var player stycoon.PlayerId
+	err := db.QueryRow("select id, password from d_user where name = ? limit 1", username).Scan(&player.Id, &password)
```

Comment:
This returns user id, not player id. You probably want to create a separate method to query player id from t_player table based on the player name from the request.

Reply:
yeah this needs more work - we should finish it in this MR - I will prepare it tomorrow ;)

Reply:
done